rpi_backlight/cli.py

```diff
@@ -97,12 +97,26 @@ def main():
         return
 
     if args.set_power:
-        if any(
-            (args.get_brightness, args.set_brightness, args.get_power, args.duration)
-        ):
-            parser.error("-p/--set-power must be used without other options")
-        backlight.power = True if args.set_power == "on" else False
+        if any((args.get_brightness, args.set_brightness, args.get_power)):
+            parser.error("-p/--set-power may only be used with -d/--duration")
+        if args.set_power == "toggle":
+            if backlight.power:
+                with backlight.fade(duration=args.duration):
+                    backlight.brightness = 0
+                if args.board_type == BoardType.RASPBERRY_PI:
```

Comment:
Sorry, my suggestion from earlier was wrong, but this won't work either - let's just change it back then.

```suggestion
                if args.board_type == "raspberry-pi":
```

Reply:
Why wont it work? Is this also considered a magic variable?